src/rules/css/detectBadMozBinding.js

```diff
@@ -2,16 +2,17 @@ import * as messages from 'messages';
 import { LOCAL_CSS_URL } from 'regex';
 
 
-export function detectBadMozBindingURL(rule) {
+export function detectBadMozBindingURL(rule, filename=null,
```

Comment:
Can we take this opportunity to rename `rule` to `cssRule`? This'll help save some potential confusion about what we're talking about since there are cssRules and rule functions.